efi-stub/boot.c

```diff
@@ -401,7 +402,7 @@ efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *_table)
 		 * bootloader name to be used. Fall back to the default bootloader
 		 * as specified in config.h
```

Comment:
This comment is outdated with your changes, isn't it?

Reply:
Yes, will remove them.